src/emu/sound.h

```diff
@@ -740,6 +740,7 @@ class sound_manager
 	static constexpr u8 MUTE_REASON_UI = 0x02;
 	static constexpr u8 MUTE_REASON_DEBUGGER = 0x04;
 	static constexpr u8 MUTE_REASON_SYSTEM = 0x08;
+	static constexpr u8 MUTE_REASON_UNTHROTTLE = 0x10;
```

Comment:
I don’t think this is necessary – you can use `ui_mute` since that’s there for the UI manager’s use while the machine is running.